### ISSUE TITLE
Building for arm: constant 2147483648 overflows int

### ISSUE THREAD

**Issue by mholt** [klauspost/compress#139]
When building a project that uses this as a dependency, I get this error:

```
$ CGO_ENABLED=1 CC=arm-linux-musleabihf-gcc GOOS=linux GOARCH=arm go build
# github.com/klauspost/compress/fse
../../../../klauspost/compress/fse/compress.go:22:13: constant 2147483648 overflows int
../../../../klauspost/compress/fse/fse.go:130:25: constant 2147483648 overflows int
```

Similarly in zstd/frameenc.go:

```
../../../../klauspost/compress/zstd/frameenc.go:108:11: constant 4294967295 overflows int
```

**Comment by klauspost** [comment]
Are you using the latest version?

It should be fixed with #129 and the value resolves to `2147483647` now.

**Comment by mholt** [comment]
Oh... good point, probably not. On further inspection it's because of a mixup between my GO111MODULE variable and an older copy of this lib that was in my GOPATH that was being used instead. My mistake! Thank you!
